.github/workflows/cherry-pick.yml

```diff
@@ -39,4 +39,54 @@ jobs:
           git cherry-pick -x ${{ github.event.inputs.shas }}
           git push --set-upstream https://github.com/ampproject/amphtml.git cherry-pick-${{ github.event.inputs.amp-version }}
```

Comment:
I just realized this needs to be built and uploaded before promoting

Reply:
Hmm... we could just add an arbitrary 20 minutes delay, which is a few more minutes than how long it usually takes to build on CircleCI and upload the artifacts

We should also add monitoring to all the workflows but that can wait until we're done with coding it :)

Reply:
How are release branches being built right now? Is it a circleci config?

Reply:
Ok I see it. I'm gonna do some exploratory work to see if it's easy enough to move to a GitHub action

Reply:
> Ok I see it. I'm gonna do some exploratory work to see if it's easy enough to move to a GitHub action

You mean move the release builds to a GH action?
I vaguely remember that we tried that at first but builds on GH took twice as long. Maybe things changed, lmk what you find!

Reply:
Oh darn. But hey this looks promising 😁  https://circleci.com/blog/trigger-circleci-pipeline-github-action/

Reply:
oh-no.jpg